README.md

```diff
@@ -20,9 +20,12 @@ dotnet-prompt enables developers to create and execute sophisticated AI workflow
 ### Installation
 
 ```bash
-# Install globally as a .NET tool
+# Install globally as a .NET tool (once published to NuGet)
 dotnet tool install -g dotnet-prompt
 
+# For development/testing, install from local package
+dotnet tool install --global --add-source ./src/DotnetPrompt.Cli/bin/Release DotnetPrompt.Cli
```

Comment:
This doesn't work. You need to run dotnet pack first. Rather than adding this line. Add a how to build and test section to the readme file